unittests/good_mock_server.py

```diff
@@ -129,6 +137,14 @@ def get_object(obj_id):
         if app.config["auth_type"]=="passport" and ("accept" in header_content and header_content["accept"] in [accept_type, "*/*"]):
             try:
                 request_body = request.get_json()
+
+                if "expand" in request_body:
+                    #if request_body["expand"].lower() == "true":
+                    #    expand = True
+                    expand = request_body["expand"]
```

Comment:
We should probably add a step to make sure expand variable is binary and not string. Also, I would remove the commented code in line 142, 143.

Reply:
Good call. Ill take a look at making these changes